Source/engine/render/dun_render.hpp

```diff
@@ -263,9 +263,10 @@ string_view MaskTypeToString(MaskType maskType);
  * @param levelCelBlock The MIN block of the level CEL file.
  * @param maskType The mask to use,
  * @param lightTableIndex The light level to use for rendering (index into LightTables).
+ * @param tbl Optional TRN for a tile.

```

Comment:
Perhaps we should remove the `lightTableIndex` and simply always pass a non-null `tbl`?